docs/index.rst

```diff
@@ -6,10 +6,23 @@
 Welcome to METASPACE Converter's documentation!
 ===============================================
 
+|Tests badge|
+|Docs badge|
+|PyPI badge|
+
 Functions to convert `METASPACE`_ datasets to `AnnData`_ objects.
 
 This makes it easy to work with the `ScanPy`_, `SquidPy`_ and `SpatialData`_ ecosystem with METASPACE data.
 
+Installation
+------------
+You can install the package via pip:
+
+.. testcode::
```

Comment:
The directive `testcode` is from the Sphinx extension [doctest](https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html#directive-testcode), as configured [here](https://github.com/metaspace2020/metaspace-converter/blob/9c8e1bb235a5ee0aee45e495ee55df8ec8c55ed8/docs/conf.py#L34). It is derived from the `code` directive and runs the Python code with pytest. Because of this, the doctests fail. It should be ignored if you replace it by just `code`.